playbooks/amazon.yml

```diff
@@ -114,5 +114,7 @@
 
 - import_playbook: ssh-setup.yml
 - import_playbook: cloud-status.yml
+- import_playbook: python.yml
```

Comment:
I was tearing my hair out trying to find this one bit of magic--I couldn't get the control flow right. Whew.

Reply:
Yeah that's originally why I used raw commands since python.yml will be included as part of streisand.yml, and I wanted to avoid doing things twice, kind of a necessary evil.